spec/scry/symbol_spec.cr

```diff
@@ -19,39 +19,39 @@ module Scry
       text_document = TextDocument.new("uri", ["class Test; end"])
```

Comment:
Any reason we are updating symbol specs in this PR?  Seems unrelated.

Reply:
Because of the way response message https://github.com/crystal-lang-tools/scry/pull/46/files#diff-82346a49d1a2a5c31d9a3445d7a9127dR9  I had to add a single CompletionItem as a response for the completion resolver.
The symbol specs expect an array in the response.